libvast/vast/accountant/config.hpp

```diff
@@ -41,6 +41,13 @@ struct accountant_config {
   self_sink self_sink;
   file_sink file_sink;
   uds_sink uds_sink;
+
+  template <class Inspector>
+  friend auto inspect(Inspector& f, accountant_config& x) {
+    f(caf::meta::type_name("vast::system::accountant_config"),
```

Comment:
```suggestion
    f(caf::meta::type_name("vast.system.accountant_config"),
```

We don't have a consistent style here throughout the code base, but this I think we use most often.